src/typescript/providers/src/wix/config.interface.ts

```diff
@@ -0,0 +1,47 @@
+import type {Variable} from 'ddns-base';
+
+import type {ProviderBaseConfig} from '../provider.interface';
+
+export interface WixProviderConfig extends ProviderBaseConfig {
+    /**
+     * The account ID linked to the API key used when updating DNS records on Wix.
+     * @example {"from": "Env", "name": "WIX_ACCOUNT_ID"}
+     */
+    accountId: Variable;
```

Comment:
Where can this be found?

The _linked_ in the JSDoc is causing some confusion on my end, is there a linking step? Could this be rewritten to be more clear?